packages/kit/src/core/adapt/builder.js

```diff
@@ -192,6 +193,14 @@ export function create_builder({
 			});
 		},
 
+		generateCspHeaderValue() {
+			if (!config.kit.csp.directives) return null;
+
+			const csp = new Csp(config.kit.csp, { prerender: false });
```

Comment:
Should this be `generatePrerenderedCspHeaderValue()` instead and `prerender` should be `true`? In the Cloudflare adapter case, it's primarily being used when serving prerendered pages.

> When mode is 'auto', SvelteKit will use nonces for dynamically rendered pages and hashes for prerendered pages. Using nonces with prerendered pages is insecure and therefore forbidden.

https://svelte.dev/docs/kit/configuration#csp

Reply:
Yeah I think you're right, but I don't think it would matter as the CSP instance here will not be taking any script or style tags as input (unless prerendered pages are analyzed and their hashes are added to it).
Also on second thought the cloudflare _headers csp would need to allow unsafe-inline for script and style directives unless we do that analysis...
At that point I'm not sure if this PR is worth it 🤔

Reply:
I'll close this PR for now to keep the queue clean but let me know if you want me to re-open it.